index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Project3</title>
+  </head>
+  <body>
+    <header>
+      <h1>
+        my conference header
+      </h1>
+    </header>
+    <nav>
+      <ul>
+        <li><a href="#About">About</a></li>
+        <li><a href="#Schedule">Schedule</a></li>
+        <li><a href="#Speakers">Speakers</a></li>
+        <li><a href="#Register">Register</a></li>
+      </ul>
+    </nav>
+    <main>
+      <p id="About">
```

Comment:
Instead of using `p` elements here, consider putting your `id`s on `section` elements and then having one or more `p` elements inside of that?